### PR TITLE
Add configuration snippet to blunderbuss, cat, cherrypick, …

### PR DESCRIPTION
**Pull request by matthyx**
…dco, golint and goose
relates to https://github.com/kubernetes/test-infra/issues/9079

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]

File: prow/plugins/heart/heart.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
 		},
 	})
 	if err != nil {
-		logrus.WithError(err).Warn("cannot generate comments for heart plugin")
+		logrus.WithError(err).Warnf("cannot generate comments for heart plugin")
```
Comment (cjwagner) [inline prow/plugins/heart/heart.go:60]: Did you mean to user `%s` and `PluginName` here and in the files below?

**Review by cjwagner — APPROVED** [review]
/hold

File: prow/plugins/heart/heart.go (hunk 1 above)
Comment (matthyx) [reply prow/plugins/heart/heart.go]: Indeed, my bad... I really should be more focussed.

Comment (cjwagner) [reply prow/plugins/heart/heart.go]: No worries.
/lgtm
/approve
/hold cancel

**Pull request merged**